server/athenian/api/preloading/cache.py

```diff
@@ -452,5 +459,48 @@ def get_memory_cache_options() -> Dict[str, Dict[str, Dict[str, List[Instrumente
                     PrecomputedGitHubReleaseMatchTimespan.acc_id,
                 ],
             },
+            PCID.done_pr_facts.value: {
```

Comment:
Interesting, I've just realized that I preloaded the table for done facts, but I actually need open facts, but test didn't fail... 🤔

Reply:
Ah ok, it's because the tests are calling the classmethod directly which is always pointing to the non-preloaded version, and probably there's no test that calls the `/metrics/prs` endpoint that makes it call the preloaded implementation.

Reply:
We should cover this with tests then. This stuff is critical.

Reply:
Totally agree, do you mind if I fix this and then I address all the tests separately? I guess this is not the only case.

Reply:
Sure.

Reply:
Thanks!